src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -3273,7 +3273,14 @@ def create_image_version(cmd, resource_group_name, gallery_name, gallery_image_n
         cred, _, _ = profile.get_login_credentials(resource=resource,
                                                    aux_subscriptions=aux_subscriptions)
         _, _, _, external_tokens = cred.get_all_tokens('https://management.azure.com/.default')
-        external_bearer_token = external_tokens[0][0] + ' ' + external_tokens[0][1]
+        if external_tokens:
+            external_token = external_tokens[0]
+            if len(external_token) >= 2:
+                external_bearer_token = external_token[0] + ' ' + external_tokens[1]
```

Comment:
```suggestion
                external_bearer_token = external_token[0] + ' ' + external_token[1]
```
Should it be `external_token[1]`?